package-lock.json

```diff
@@ -182,12 +182,12 @@
       }
```

Comment:
The Dependency Review status check failed with this error:
`Error: $GITHUB_STEP_SUMMARY upload aborted, supports content up to a size of 1024k, got 1542k. For more information see: https://docs.github.com/actions/using-workflows/workflow-commands-for-github-actions#adding-a-markdown-summary`

I suspect that's because it tries to print a huge report that includes all the added and removed dependencies. This seems innocuous so we can ignore this issue and complete the PR. @rajsite if you agree, feel free to complete it.

There is an issue tracking that bug which they marked as fixed in 4.3.3 but people have been commenting otherwise: https://github.com/actions/dependency-review-action/issues/774 I can't tell what version we're using other that `@v4`

Reply:
Or is it complaining about this high sev vulnerability? 
[Server-Side Request Forgery in axios](https://github.com/advisories/GHSA-8hc4-vh64-cxmj) [(GHSA-8hc4-vh64-cxmj)](https://github.com/advisories/GHSA-8hc4-vh64-cxmj)
    High severity
    Patched version: 1.7.4

Reply:
I say we abandon this PR in favor of addressing the corresponding actual dependency updates: https://github.com/ni/nimble/pull/2365

I have a suspicion that there are too many diffs in the PR anyway and the lock file should be rebuilt but that will also likely happen addressing that other change.

Reply:
This branch also results in a dirty package lock if you run a local install. I started from this PR and made some more changes in #2370 to address the above issues. I think we should abandon this one.

Reply:
Gonna close this PR to address in https://github.com/ni/nimble/pull/2367 instead